src/main/js/field/EndpointField.js

```diff
@@ -6,6 +6,7 @@ import LabeledField from 'field/LabeledField';
 import * as FieldModelUtilities from 'util/fieldModelUtilities';
 import { createNewConfigurationRequest } from 'util/configurationRequestBuilder';
 import { connect } from 'react-redux';
+import StatusMessage from "./StatusMessage";
```

Comment:
This import path is incorrect. Please make this an absolute path like the other imports